src/js/libgeo/GeoOps.js

```diff
@@ -170,6 +170,30 @@ geoOps.Vertical.updatePosition = function(el) {
 };
 
 
+geoOps.LineByFixedAngle = {};
+geoOps.LineByFixedAngle.kind = "L";
+geoOps.LineByFixedAngle.initialize = function(el) {
+    var a = CSNumber._helper.input(el.angle);
+    var c = CSNumber.cos(a);
+    var s = CSNumber.sin(a);
+    // Setup matrix for applying the angle rotation
+    el.rot = List.turnIntoCSList([
+        List.turnIntoCSList([c, CSNumber.neg(s), CSNumber.zero]),
+        List.turnIntoCSList([s, c, CSNumber.zero]),
+        List.turnIntoCSList([CSNumber.zero, CSNumber.zero, CSNumber.zero])
```

Comment:
Hi just a quick remark: i was a bit confused why one would use CSNumber.zero in el.rot(3,3). Of course we only deal with far points here, but mathematically this isn't even a projective transformation. 

The rest looks fine to me.